README-dev.md

```diff
@@ -30,10 +30,19 @@ For most users, building o1js is as simple as running:
 
 ```sh
 npm install
+gh auth login #gh is used to download the compiled artifacts
 npm run build
 ```
 
-This command compiles the TypeScript source files, making them ready for use. The compiled OCaml and WebAssembly artifacts are version-controlled to simplify the build process for end users. These artifacts are stored under `src/bindings/compiled` and contain the artifacts needed for both node and web builds. These files only have to be regenerated if there are changes to the OCaml or Rust source files.
+This command downloads the artifacts from github if they are missing and compiles the TypeScript source files, making them ready for use.
```

Comment:
Can we add a section explicitly directed at external contributors that is easy to find for new developers that want to contribute to o1js?

Reply:
I added subsections for external and internal contributors, I think it reads better now.